cpp/src/arrow/util/thread-pool-benchmark.cc

```diff
@@ -168,6 +168,8 @@ static void WorkloadCost_Customize(benchmark::internal::Benchmark* b) {
     b->Args({w});
   }
   b->ArgNames({"task_cost"});
+  b->UseRealTime();
+  b->Repetitions(1);
```

Comment:
Is it intended to force the number of repetitions?

Reply:
Yes, wanted to minimize the time it takes to run and this one was in the heavy. I'll remove it since it doesn't change by that much.